### DISCUSSION TITLE
咋进不去服务器

### DISCUSSION THREAD

**Discussion by E0x72-21** [Meloong-Git/PCL#4976] [提问] [answered]
我自己的服务器，服务器核心是fabric
![image](https://github.com/user-attachments/assets/fed517b4-3795-4afc-847f-671b275ea989)

[disconnect-2024-10-18_21.41.56-client.txt](https://github.com/user-attachments/files/17442324/disconnect-2024-10-18_21.41.56-client.txt)

模组列表：
architectury-13.0.6-fabric.jar
badhorsefix-2.1.0-fabric.jar
BadOptimizations-2.1.4-1.21.jar
baritone-1.21.1-20240826.213754-1.jar
bedrock-miner-1.21-1.2.21.jar
calcmod-1.3.2+fabric.1.21.jar
caxton-0.6.0-alpha.3+1.21.1-FABRIC.jar
cicada-lib-0.8.3+1.21-and-above.jar
clientcommands-2.9.3.jar
cloth-config-15.0.140-fabric.jar
continuity-3.0.0-beta.5+1.21.jar
do_a_barrel_roll-fabric-3.7.2+1.21.jar
durabilitytooltip-1.1.5-fabric-mc1.21.jar
early-loading-screen-0.8.0.jar.disabled
EarlyLoadingBar-1.0.jar.disabled
emi-1.1.16+1.21.1+fabric.jar
exordium-fabric-1.3.0-mc1.21.jar
fabric-api-0.106.0+1.21.1.jar
fabric-language-kotlin-1.12.3+kotlin.2.0.21.jar
fast-ip-ping-v1.0.5-mc1.21.1-fabric.jar
fastpaintings-1.21-1.2.13-fabric.jar.disabled
GpuTape-1.18x-1.21x-1.0.3.jar
idle_boost-fabric-1.2.0+1.21.1.jar
ImmediatelyFast-Fabric-1.2.21+1.21.1.jar
iris-1.7.3+mc1.21.jar
Ksyxis-1.3.2.jar
libIPN-fabric-1.21-6.0.3.jar
malilib-fabric-1.21.1-0.21.0.jar
meteor-client-0.5.8-2167.jar
meteor-crash-addon-0.6.jar
meteor-plus-1.21.1_1.0.9.0.jar
meteor-rejects-addon-0.3.jar
midnightlib-fabric-1.6.3.jar
moonlight-1.21-2.14.32-fabric.jar.disabled
noisium-fabric-2.3.0+mc1.21-1.21.1.jar
nomenublur-1.0.0.jar
nvidium-0.3.1.jar
offers-hud-1.21-1.6.1.jar
placeholder-api-2.4.1+1.21.jar
ScalableLux-0.1.0+fabric.26c6e72-all.jar
Searchables-fabric-1.21.1-1.0.1.jar
seedcrackerX-2.15.0.jar
smoothskies-2.0.4+1.20.4-fabric.jar
supermartijn642configlib-1.1.8-fabric-mc1.21.jar
threadtweak-fabric-0.1.5+mc1.21.1.jar
threatengl-fabric-1.3.3-release.1.jar
tweakermore-v3.21.0-mc1.21.1.jar.disabled
tweakeroo-fabric-1.21.1-0.21.50.jar
ViaFabricPlus-3.4.8.jar
villagertimetable-1.0.0.jar
vmp-fabric-mc1.21.1-0.2.0+beta.7.168-all.jar
world_preview-fabric-1.21-1.3.1.jar
YetAnotherConfigLib-3.5.0+1.21-fabric.jar
skinlayers3d-fabric-1.6.7-mc1.21.jar
XaerosWorldMap_1.39.0_Fabric_1.21.jar
XaeroPlus-2.23.2+fabric-1.21-WM1.39.0-MM24.4.0.jar
Xaeros_Minimap_24.5.0_Fabric_1.21.jar
InventoryProfilesNext-fabric-1.21-2.0.6.jar
dynamic-fps-3.7.3+minecraft-1.21.0-fabric.jar
entityculling-fabric-1.7.0-mc1.21.jar
notenoughcrashes-4.4.8+1.21-fabric.jar
smoothscroll-2.1.3+1.21.jar
screenshot-to-clipboard-fabric-1.0.10.jar
litematica-fabric-1.21.1-0.19.50.jar
BetterF3-11.0.2-Fabric-1.21.1.jar
mcwifipnp-1.7.3-1.21-fabric.jar
cullleaves-fabric-3.4.0.jar
modmenu-11.0.3.jar
dark-loading-screen-1.6.14.jar
shulkerboxtooltip-fabric-5.1.1+1.21.1.jar
inventoryhud.fabric.1.21.1-3.4.26.jar
itemscroller-fabric-1.21.1-0.24.50.jar
Jade-1.21.1-Fabric-15.8.0.jar
modernfix-fabric-5.19.3+mc1.21.1.jar
voicechat-fabric-1.21.1-2.5.24.jar
axolotl-item-fix-1.1.9.jar
permanent-chat-logs-0.2.3-build20240714.jar
PresenceFootsteps-1.10.0+1.21.jar
appleskin-fabric-mc1.21-3.0.5.jar
IMBlocker_4.0.9b+1.21.jar
minihud-fabric-1.21.1-0.32.50.jar
sodium-extra-0.5.7+mc1.21.jar
sodium-fabric-0.5.11+mc1.21.jar
ferritecore-7.0.0-fabric.jar
indium-1.0.35+mc1.21.jar
lithium-fabric-0.14.0-snapshot+mc1.21.1-build.88.jar
Controlling-fabric-1.21.1-19.0.3.jar
music_control-1.8.3+1.21.jar

**Comment by tangge233** [comment]
TweakerMore 发送了错误的包

**Reply by E0x72-21** [reply]
为什么会这样呢？（

**Reply by tangge233** [reply]
IDK，错误日志只能知道哪里出了问题。解决还需要找模组作者……

**Reply by E0x72-21** [reply]
好吧，禁用了tweaker more还是这样
[disconnect-2024-10-18_23.23.27-client.txt](https://github.com/user-attachments/files/17442665/disconnect-2024-10-18_23.23.27-client.txt)

**Reply by tangge233** [reply]
只剩下 netty 在爆炸了……

**Reply by E0x72-21** [reply]
> 只剩下 netty 在爆炸了……

啥玩意？gpt的回答我也是听君一席话如听一席话
![image](https://github.com/user-attachments/assets/80f96768-afbb-4fd5-b3d1-4b318ef287dc)

**Reply by tangge233** [reply]
https://www.mcmod.cn/class/16197.html
你看看使用这个后可不可以

**Reply by E0x72-21** [reply]
> https://www.mcmod.cn/class/16197.html 你看看使用这个后可不可以

还是炸啊（恼

**Reply by tangge233** [reply]
怪 
Caused by: java.lang.IndexOutOfBoundsException: readerIndex(25) + length(1) exceeds writerIndex(25): PooledUnsafeDirectByteBuf(ridx: 25, widx: 25, cap: 25)
数据包发来的长度只有 1 字节
要不重新部署下服务端试试吧，我没得办法了……

**Reply by E0x72-21** [reply, marked as answer]
> 怪 Caused by: java.lang.IndexOutOfBoundsException: readerIndex(25) + length(1) exceeds writerIndex(25): PooledUnsafeDirectByteBuf(ridx: 25, widx: 25, cap: 25) 数据包发来的长度只有 1 字节 要不重新部署下服务端试试吧，我没得办法了……

目前找到的嫌疑模组：玉，moonlight lib最新版

**Reply by Minemetero** [reply]
Netty炸把Netty给禁用了

**Comment by Minemetero** [comment]
meteor-client-0.5.8-2167.jar
meteor-crash-addon-0.6.jar
meteor-plus-1.21.1_1.0.9.0.jar
meteor-rejects-addon-0.3.jar

居然是外挂玩家

**Reply by E0x72-21** [reply]
bbtt玩家怎么你了（
